ui/components/app/modals/turn-on-metamask-notifications/turn-on-metamask-notifications.tsx

```diff
@@ -51,14 +51,14 @@ export default function TurnOnMetamaskNotifications() {
   );
   const isProfileSyncingEnabled = useSelector(selectIsProfileSyncingEnabled);
 
-  const [buttonState, setButtonState] = useState<boolean>(
+  const [loadingState, setLoadingState] = useState<boolean>(
```

Comment:
I'm always a bit more in favor of "boolean" names, like `[isLoading, setIsLoading]`, wdyt?

Reply:
Yep, you are right. I was a bit lazy :)